src/inputType/PanInput.ts

```diff
@@ -301,23 +307,24 @@ export class PanInput implements InputType {
       ]
     );
     this._observer.release(this, prevEvent, velocity);
+    activeInput.prevEvent = null;
```

Comment:
`activeInput` inside `PanInput` looks quite awkward for me, wasn't there any better naming for that?

Reply:
How about using name like `activeEvent` ? 
This means whether the type of event currently handling input is pointer, touch, or mouse.

Reply:
That would be better 👍